policy/handler.go

```diff
@@ -309,6 +310,12 @@ func (h *Handler) updateHandler(current, next *sdk.ScalingPolicy) {
 	// policy's evaluation interval has changed.
 	if current == nil || current.EvaluationInterval != next.EvaluationInterval {
 		h.ticker.Stop()
+
+		// Add a small random delay to spread the first evaluation of policies
+		// that are evaluated at the same time.
```

Comment:
It might be because it's early; I feel it would be nice for the comment to detail what the delay range could be to save the reader from having to work this out themselves.